zbus/src/match_rule/builder.rs

```diff
@@ -1,4 +1,4 @@
-use std::convert::{TryFrom, TryInto};
+use std::convert::TryInto;
```

Comment:
Now that we've `2021` edition enabled, you can just drop this whole line I think.

Reply:
Awesome